docs/source/topics/frontera-settings.rst

```diff
@@ -437,6 +437,96 @@ Default: ``timedelta(days=1)``
 Time between document visits, expressed in ``datetime.timedelta`` objects. Changing of this setting will only affect
 documents scheduled after the change. All previously queued documents will be crawled with old periodicity.
 
+.. _cassandra-settings:
+
+Cassandra
+---------
+
+
+.. setting:: CASSANDRABACKEND_DROP_ALL_TABLES
+
+CASSANDRABACKEND_DROP_ALL_TABLES
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``False``
+
+Set to ``True`` if you need to drop of all DB tables on backend instantiation (e.g. every Scrapy spider run).
```

Comment:
I would remove `(e.g. every Scrapy spider run)`, because it shouldn't be connected with Scrapy run. Different run modes are requiring backend to instantiate in various processes.